content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx

```diff
@@ -28,37 +28,38 @@ Once you're done with that guide, you can choose **one of the following four gui
 
 > **Note**: Please [let us know](https://github.com/prisma/p1-to-p2-upgrade-path-feedback/issues/new) if your upgrade scenario is not covered by these guides.
 
-## Main differences between Prisma 1 and Prisma 2
+## Main differences between Prisma 1 and Prisma 2+
 
-On a high-level, the biggest differences between Prisma 1 and Prisma 2 are the following:
+On a high-level, the biggest differences between Prisma 1 and Prisma 2+ are the following:
 
-- Prisma 2 doesn't require hosting a database proxy server (i.e., the [Prisma server](https://www.prisma.io/docs/1.34/prisma-server/)).
-- Prisma 2 makes the features of Prisma 1 more modular and splits them into dedicated tools:
+- Prisma 2+ doesn't require hosting a database proxy server (i.e., the [Prisma server](https://www.prisma.io/docs/1.34/prisma-server/)).
+- Prisma 2+ makes the features of Prisma 1 more modular and splits them into dedicated tools:
   - Prisma Client: An improved version of Prisma client 1.0
   - Prisma Migrate: Data modeling and migrations (formerly `prisma deploy`).
 - Prisma 1 datamodel and `prisma.yml` have been merged into the [Prisma schema](/concepts/components/prisma-schema).
-- Prisma 2 uses its own [modeling language](https://github.com/prisma/specs/tree/master/schema) instead of being based on GraphQL SDL.
-- Prisma 2 doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. This means Prisma 2 [doesn't support Prisma binding](https://www.prisma.io/docs/more/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
-- More powerful introspection allows connecting Prisma 2 to any existing database
+- Prisma 2+ uses its own [modeling language](https://github.com/prisma/specs/tree/master/schema) instead of being based on GraphQL SDL.
+- Prisma 2+ doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. 
+  - This means Prisma 2+ [doesn't support Prisma binding](https://www.prisma.io/docs/more/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
```

Comment:
Unrelated change to make this bullet point separate from the more general main bullet.